recipes-ni/ni-hw-support/ni-hw-support-common_1.0.bb

```diff
@@ -0,0 +1,54 @@
+SUMMARY = "Support scripts and utilities common to all NI hardware products"
```

Comment:
I'm concerned about the package name and the direction it takes when you add a hardware family-specific recipe like ni-hw-support-crio, which sounds like a driver package to me and can be confused for one. There are other hwsupport packages in the feed that provide driver support.

Reply:
Granted. I'm trying to use this family to isolate files which enable NI hardware *as opposed to* generic x64 PC hardware. Do you have an alternative namespace in mind?

Reply:
ni-hwutils? ni-hw-utils? ni-hw-support-utils? Do these names support my case at all? :)